File: Faza6/OnlineGamesShop/src/main/java/com/rich/onlinegamesshop/model/reports/MonthlyGameStatisticsReport.java
package com.rich.onlinegamesshop.model.reports;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MonthlyGameStatisticsReport {
    private String genre;
    private String publisher;
    private BigDecimal monthNumber;
    private long totalReviews;
    private BigDecimal totalRevenue;

}

File: Faza6/FrontEnd/Angular/src/app/model/Reports/MonthlyGameStatisticsReport.ts
import Big from "big.js";
export class MonthlyGameStatisticsReport {
  genre:string;
  publisher:string;
  monthNumber:Big;
  totalReviews:number;
  totalRevenue:Big;
  constructor(
    genre:string,
    publisher:string,
    monthNumber:Big,
    totalReviews:number,
    totalRevenue:Big,
  ){
    this.genre=genre;
    this.publisher=publisher;
    this.monthNumber=monthNumber;
    this.totalReviews=totalReviews;
    this.totalRevenue=totalRevenue;
  }
}
